packages/components/src/unit-control/types.ts

```diff
@@ -0,0 +1,38 @@
+export interface WPUnitControlUnit {
+	/**
+	 * The value for the unit, used in a CSS value (e.g `px`).
+	 */
+	value: string;
```

Comment:
Should we look into typing the allowed units more strictly? I'm not fully aware if users of `UnitControl` are allowed to specify just about any unit they want.

Reply:
That's a good question. I think when it comes to the value in `WPUnitControlUnit` a case can be made for allowing arbitrary values so that the UnitControl component can be used for units that we haven't factored in (or thought of). An example came to mind — if someone were to develop a Map block for real estate listings or something of that nature, they might want to include a UnitControl for adjusting the search radius (or zoom level) in the map. In that case, perhaps they'd use units like `mi` and `km` for distances.

I see there's already a test for custom unit values, but I've added in an additional storybook story that includes custom units for `km`, `mi`, `m`, and `yd` as an example, so that we've got it in the component.

Reply:
Thank you for the explanation, and for adding a Storybook example. Let's leave it as a more relaxed `string` type